test/specs/kefir-observable.js

```diff
@@ -0,0 +1,49 @@
+let {Kefir} = require('../test-helpers.coffee');
+
+describe('Kefir.Observable', function() {
+    describe('observe', function() {
+        let em, count, obs, sub;
```

Comment:
Not sure if we can use es6 here. I mean I'm not sure it's transpiled, so if we use es6 then tests may work only in modern environments.